gormigrate.go

```diff
@@ -46,11 +46,17 @@ type Gormigrate struct {
 	initSchema InitSchemaFunc
 }
 
+// MigrationRecord in migrations table.
+type MigrationRecord struct {
+	ID        int
+	Migration string
+}
+
 var (
 	// DefaultOptions can be used if you don't want to think about options.
 	DefaultOptions = &Options{
 		TableName:      "migrations",
-		IDColumnName:   "id",
+		IDColumnName:   "migration",
```

Comment:
This will break things for who already use the lib. Let's keep `id` as the default name.